packages/url_launcher/url_launcher_platform_interface/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: url_launcher_platform_interface
+description: A common platform interface for the url_launcher plugin.
+author: Flutter Team <flutter-dev@googlegroups.com>
+homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher_platform_interface
+version: 0.1.0
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  url_launcher:
+    path: ../url_launcher
```

Comment:
Should this be a dependency on a published version of the plugin, instead of a path?

Reply:
I think we shouldn't have this dependency here at all (see my comment on the test_

Reply:
Removed dependency